lua/lsp-status/statusline.lua

```diff
@@ -110,7 +112,7 @@ local function get_lsp_statusline(bufnr)
     end
   end
 
-  if base_status ~= '' then return symbol .. base_status .. ' ' end
+  if not config.diagnostics or base_status ~= '' then return symbol .. base_status .. ' ' end
```

Comment:
Is this modified condition correct? I'm not sure that this is necessary.

Reply:
Yes, this is necessary to avoid return with `indicator_ok`.

Reply:
Yes, but in the case that you have diagnostics disabled, but an empty status string otherwise, doesn't this cause an erroneous return value?

Reply:
No, it works as expected.

Reply:
Actually, in my testing, it does have the issue I mentioned - the return value has a trailing space that shouldn't be there. If we're not outputting `indicator_ok`, we should just output `symbol` with no trailing space, I think.

Reply:
Got it, done!

Reply:
Thanks, looks good!